airflow/config_templates/config.yml

```diff
@@ -386,13 +386,13 @@
       type: string
       default: "0o077"
       example: ~
-    - name: dataset_event_manager_class
+    - name: dataset_manager_class
       description: Class to use as dataset event manager.
```

Comment:
```suggestion
      description: Class to use as dataset manager, including dataset events.
```

Reply:
I think I'm 👎 on this specific one.

Reply:
I did just commit the "dataset event manager" -> "dataset manager" change though.

Reply:
:+1: Now if I can find the way to dismiss this suggestion...